pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -2053,8 +2053,8 @@ def label(self, decimals=None, base_label=None, cache=None):
     def _controlled(self, _):
         return Barrier(wires=self.wires)
 
-    def adjoint(self, do_queue=False):
-        return Barrier(wires=self.wires)
+    def adjoint(self, do_queue=True):
```

Comment:
How come `do_queue` was changed for some ops?

Reply:
For Snapshot and Barrier the argument wasn't doing anything as the class is instantiated without passing along the argument:
```py
        return Barrier(wires=self.wires)
```
 I changed the default to True because that is the existing behaviour given the class was instantiated without passing `do_queue` and the default value for it in `__init__` is True.

Reply:
But the plan is to remove the `do_queue` keyword entirely in the future.